scrapers_next/la/committees.py

```diff
@@ -41,19 +42,31 @@ def process_item(self, item):
 
 
 class CommitteeDetail(HtmlPage):
+
+    jr_sr_regex = re.compile(r"(.*)\s+(Jr|Jr\.|Sr|Sr\.)\s+(.*)")
```

Comment:
Would you mind adding in a comment on what the regex does? Its not too hard to figure it out by reading the code but the name jr_sr_regex on its own isnt immediately obvious.

Reply:
[Fixed](https://github.com/openstates/openstates-scrapers/pull/4596/commits/34ff70a6976ad02a53c42e2f0cdcf8777239b282).